### DISCUSSION TITLE
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db_formsinar.model_has_roles'

### DISCUSSION THREAD

**Discussion by KodokMagang** [spatie/laravel-permission#2709] [Q&A]
I am experiencing a confusing error, because I did not delete the model_has_role table at all but there is an error log like this “SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db_formsinar.model_has_roles' doesn't exist (Connection: mysql, SQL: select `roles`.*, `model_has_roles`.`model_id` as `pivot_model_id`, `model_has_roles`.`role_id` as `pivot_role_id`, `model_has_roles`. `model_type` as `pivot_model_type` from `roles` inner join `model_has_roles` on `roles`.`id` = `model_has_roles`.`role_id` where `model_has_roles`.`model_id` in (1) and `model_has_roles`.`model_type` = App\Models\User)” how to restore the table? Please help me.

**Comment by ram2228** [comment]
Try migrate the table again
